daft/expressions/expressions.py

```diff
@@ -3461,3 +3466,43 @@ class ExpressionEmbeddingNamespace(ExpressionNamespace):
     def cosine_distance(self, other: Expression) -> Expression:
         """Compute the cosine distance between two embeddings"""
         return Expression._from_pyexpr(native.cosine_distance(self._expr, other._expr))
+
+
+class ExpressionGeometryNamespace(ExpressionNamespace):
+    def decode(self, raise_on_error: bool = True) -> Expression:
```

Comment:
true! I had thought that makes sense and is familiar namespacing in SQL, while in DF api it would be `geo`. Otherwise `geo.st_area` feels a bit redundant, but indeed keeping same naming between sql and df api is nice. Specifically for `decode`, the one above is not exactly `st_geomfromwkb` since it does `st_geomfromtext` when the type is UTF-8, wkb if it is binary.